src/rapids_singlecell/decoupler_gpu/_helper/_pv.py

```diff
@@ -97,9 +78,7 @@ def fdr_bh_axis1_cupy_optimized(ps, *, mem_gb: float = 4.0) -> cp.ndarray:
         ps_bh = ps_sorted * scale  # (R, m) float64
 
         # 4) reverse cumulative min via custom kernel
-        ps_mon = cp.empty_like(ps_bh)
-        blocks = (R + threads - 1) // threads
-        _rev_cummin64((blocks,), (threads,), (ps_bh, ps_mon, R, m))
```

Comment:
out of curiosity: why is it possible to get rid of all the block size and thread magic numbers now?

Reply:
ah yeah, the magic numbers are still there, e.g.:

https://github.com/scverse/rapids_singlecell/blob/d38600052328f6cea29e86ac3dadef3faf878ba6/src/rapids_singlecell/_cuda/harmony/outer/outer.cu#L12-L14

but I do feel like it’s less.